spec/controllers/ctc/portal/dependents_controller_spec.rb

```diff
@@ -47,9 +47,9 @@
 
   context "#update" do
     around do |example|
-      Timecop.freeze(DateTime.new(2021, 3, 4, 5, 10))
-      example.run
-      Timecop.return
+      Timecop.freeze(DateTime.new(2021, 3, 4, 5, 10)) do
```

Comment:
am I correct in understanding that this was not causing a failure before because we were returning Timecop but this change is just a stylistic preference? Or was this `Timecop.return` failing to actually exit the freeze block before?

Reply:
Yes - all Timecop changes except for the test in `rack_attack_spec.rb` were style changes to get us consistent in using the block sytax (`Timecop.freeze do ... end`) instead of needing to remember to call Timecop.return.